docs/en/guides/getting-started.md

```diff
@@ -110,6 +110,34 @@ To simplify usage, `vue-test-utils` applies all updates synchronously so you don
 
 *Note: `nextTick` is still necessary when you need to explictly advance the event loop, for operations such as asynchronous callbacks or promise resolution.*
 
+If you do still need to use `nextTick` in your test files, be aware that any errors thrown inside it may not be caught by your test runner as it uses promises internally. There are two approaches to fixing this: either you can set the `done` callback as Vue's global error handler at the start of the test, or you can call `nextTick` without an argument and return it as a promise:
+
+```js
+// this will not be caught
+it('will time out', (done) => {
+  Vue.nextTick(() => {
+    expect(true).toBe(false);
+    done();
```

Comment:
This will fail when you run `npm run lint:docs`. 

For some reason it's not running inside CI.

Can you lint and fix the errors